assets/javascripts/controllers/resolution-ui-builder.js.es6

```diff
@@ -0,0 +1,154 @@
+import I18n from "I18n";
+import Controller from "@ember/controller";
+import discourseComputed, { observes } from "discourse-common/utils/decorators";
+import EmberObject from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { throttle } from "@ember/runloop";
+
+export default Controller.extend({
+  init() {
+    this._super(...arguments);
+    this._setupPoll();
+  },
+
+  @discourseComputed("pollOptions")
+  pollOptionsCount(pollOptions) {
+    if (pollOptions.length === 0) return 0;
```

Comment:
Do we really need this check? If `pollOptions` is 0, the code below also returns 0.